opus/import/do_param_info.py

```diff
@@ -54,6 +68,13 @@ def create_import_param_info_table():
                            f'"{category_name}/{field_name}" has RANGE type '
                            +'without numerical format')
 
+            # if pi_ranges exist in .json, get the corresponding ranges info
+            # from dict and convert it to str before storing to database
+            ranges = column.get('pi_ranges', None)
+            if ranges:
+                ranges = ranges_json.get(ranges)
```

Comment:
Use ranges_json[ranges] because we don't want to have a default.
Check to see if key is in ranges first and display an error if not.